render/fx_renderer/fx_framebuffer.c

```diff
@@ -20,6 +20,37 @@ static const struct wlr_addon_interface buffer_addon_impl = {
 	.destroy = handle_buffer_destroy,
 };
 
+void fx_framebuffer_get_or_create_bufferless(struct fx_renderer *renderer,
```

Comment:
Q: why is this function have `bufferless` in its name? This question might be from a lack of understanding of the wlr allocator but I'm curious about the naming here, seems it does create a buffer?

Reply:
The reason is that our custom fx_framebuffers aren't requesting the swapchain for a new wlr_buffer, so it's “bufferless”. I agree that the naming isn't the best, any input on a new name?